bitnami/cert-manager/templates/controller/deployment.yaml

```diff
@@ -128,3 +128,8 @@ spec:
         {{- if .Values.controller.extraVolumes }}
         {{- include "common.tplvalues.render" (dict "value" .Values.controller.extraVolumes "context" $) | nindent 8 }}
         {{- end }}
+      dnsPolicy: {{ .Values.controller.dnsPolicy }}
+      {{- with .Values.controller.dnsConfig }}
+      dnsConfig:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
I think we should use the `common.tplvalues.render` helper here as we do in other places.

Reply:
Replaced with helper function